bokeh/charts/builder/catheatmap_builder.py

```diff
@@ -17,10 +17,10 @@
 #-----------------------------------------------------------------------------
```

Comment:
As you are already doing this big change I'd change the filename to `heatmap_builder.py`. We already changed the class (Builder) name so the filename is a leftover from the previous old implementation.